resources/perf.webkit.org/public/v3/bundled-scripts.js

```diff
@@ -674,7 +674,7 @@ addMetric(metric){this._metrics.push(metric);}
 isHidden(){return this._hidden;}}
 if(typeof module!='undefined')
 module.exports.Test=Test;class Metric extends LabeledObject{constructor(id,object)
-{super(id,object);this._aggregatorName=object.aggregator;object.test.addMetric(this);this._test=object.test;this._platforms=[];const suffix=this.name().match('([A-z][a-z]+|FrameRate)$')[0];this._unit={'FrameRate':'fps','Runs':'/s','Time':'ms','Duration':'ms','Malloc':'B','Heap':'B','Allocations':'B','Size':'B','Score':'pt','Power':'W',}[suffix];}
+{super(id,object);this._aggregatorName=object.aggregator;object.test.addMetric(this);this._test=object.test;this._platforms=[];const suffix=this.name().match('([A-Z][a-z]+|FrameRate)$')[0];this._unit={'FrameRate':'fps','Runs':'/s','Time':'ms','Duration':'ms','Malloc':'B','Heap':'B','Allocations':'B','Size':'B','Score':'pt','Power':'W',}[suffix];}
```

Comment:
Huh, this is a weird typo in the built script. I wonder how that happened?

Reply:
As discussed in the meeting, this was likely from not regenerating the output after fixing it in #282 .